source/analysis/AnalysisManager.cpp

```diff
@@ -236,6 +236,46 @@ void AnalysisManager::getFunctionDrivers(const CallExpression& expr, const Symbo
         getFunctionDrivers(*call, containingSymbol, visited, drivers);
 }
 
+void AnalysisManager::getTaskTimingControls(const CallExpression& expr,
+                                            SmallSet<const SubroutineSymbol*, 2>& visited,
+                                            std::vector<const ast::Statement*>& controls) {
+    if (expr.getSubroutineKind() != SubroutineKind::Task) {
```

Comment:
You also need to skip system calls here or else you'll crash on trying to access the wrong element of the variant below.